compiler/rustc_hir_analysis/messages.ftl

```diff
@@ -85,6 +85,19 @@ hir_analysis_cmse_output_stack_spill =
     .note1 = functions with the `{$abi}` ABI must pass their result via the available return registers
     .note2 = the result must either be a (transparently wrapped) i64, u64 or f64, or be at most 4 bytes in size
 
+hir_analysis_coerce_pointee_cannot_coerce_unsize = `{$ty}` cannot be coerced to an unsized type
+    .note = `derive(CoercePointee)` demands that `{$ty}` can be coerced to an unsized type
```

Comment:
This note is not exercised by any test, I think?

Reply:
I just realised that the note cannot be exercised because the span falls within the macro expansion.

Instead it is overwritten by this message.
```
note: this error originates in the derive macro `CoercePointee` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
Maybe I need to switch to the span of the ADT definition.